lib/actions.js

```diff
@@ -1064,6 +1064,20 @@ exports.exists = function(selector) {
 };
 
 /**
+ * Determine if the selector exists, at least once, on the page  and execute function if it does
+ * @param {string} [selector]
+ * @param {function} fn
+ */
+exports.ifExists = function(selector, fn) {
+	debug('.ifExists()', selector);
+	return this.count(selector).then(function(count) {
```

Comment:
You should just use the existing `.exists()` action rather than re-implementing it.